src/collective/cover/testing.py

```diff
@@ -115,6 +115,16 @@ def generate_jpeg(width, height):
     return output
 
 
+# FIXME: workaround for https://github.com/plone/plone.app.testing/issues/39
+#        Products.TinyMCE is used only in Plone 4
+if not IS_PLONE_5:
+    autoform = ('plone.autoform', {'loadZCML': True})
+    tinymce = ('Products.TinyMCE', {'loadZCML': True})
+    products = list(PLONE_FIXTURE.products)
+    products.insert(products.index(tinymce), autoform)
+    PLONE_FIXTURE.products = tuple(products)
```

Comment:
We have to do this in ```brasil.gov.portal``` if we want to update Plone.

But honestly I think this should be fixed in plone.app.testing.

Reply:
I need to keep going on with development; we have deadlines.

Reply:
Then put it on ```brasil.gov.portal```.

Reply:
done in d530a062cbc8b9fc71c0d38fcd1c2ed23da32825